frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/general-tab/simulated-region-overview-general-tab.component.ts

```diff
@@ -1,16 +1,173 @@
+import type { MemoizedSelector } from '@ngrx/store';
+import { createSelector, Store } from '@ngrx/store';
+import type {
+    Material,
+    Patient,
+    PatientStatus,
+    Personnel,
+    PersonnelType,
+    UUID,
+    Vehicle,
+    WithPosition,
+} from 'digital-fuesim-manv-shared';
+import {
+    isInSpecificSimulatedRegion,
+    SimulatedRegion,
+} from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
 import { ExerciseService } from 'src/app/core/exercise.service';
+import type { AppState } from 'src/app/state/app.state';
+import {
+    createSelectSimulatedRegion,
+    selectMaterials,
+    selectPatients,
+    selectPersonnel,
+    selectVehicleTemplates,
+    selectVehicles,
+} from 'src/app/state/application/selectors/exercise.selectors';
+
+const patientCategories = ['red', 'yellow', 'green', 'black'] as const;
+type PatientCategories = (typeof patientCategories)[number];
+
+const personnelCategories = [
+    'gf',
+    'notarzt',
+    'notSan',
+    'rettSan',
+    'san',
+] as const;
+type PersonnelCategories = (typeof personnelCategories)[number];
 
 @Component({
     selector: 'app-simulated-region-overview-general-tab',
     templateUrl: './simulated-region-overview-general-tab.component.html',
     styleUrls: ['./simulated-region-overview-general-tab.component.scss'],
 })
-export class SimulatedRegionOverviewGeneralTabComponent {
+export class SimulatedRegionOverviewGeneralTabComponent implements OnInit {
     @Input() simulatedRegion!: SimulatedRegion;
 
-    constructor(private readonly exerciseService: ExerciseService) {}
+    public readonly vehicleTemplates$ = this.store.select(
+        selectVehicleTemplates
+    );
+
+    patients: {
+        [Key in `${PatientCategories | 'all'}$`]?: Observable<Patient[]>;
+    } = {};
+
+    vehicles$?: Observable<{ [Key in string]?: Vehicle[] }>;
+
+    personnel: {
+        [Key in `${PersonnelCategories | 'all'}$`]?: Observable<Personnel[]>;
+    } = {};
+
+    material$?: Observable<Material[]>;
+
+    public patientsCollapsed = true;
+    public vehiclesCollapsed = true;
+    public personnelCollapsed = true;
+
+    constructor(
+        private readonly exerciseService: ExerciseService,
+        private readonly store: Store<AppState>
+    ) {}
+
+    ngOnInit(): void {
+        const containedPatientsSelector =
+            this.createSelectContainedElements(selectPatients);
+        const containedPersonnelSelector =
+            this.createSelectContainedElements(selectPersonnel);
+
+        this.patients.all$ = this.store.select(containedPatientsSelector);
+        patientCategories.forEach((category) => {
+            this.patients[`${category}$`] = this.store.select(
+                this.createSelectByPredicate(
+                    containedPatientsSelector,
+                    this.createPatientStatusPredicate(category)
+                )
+            );
+        });
+
+        this.vehicles$ = this.store.select(
```

Comment:
This would require handling the `Observable` of the templates correctly (what should happen if the templates change?). Even if there is no way the templates could change during an exercise at the moment, the code would still be more complex since `store.select` only gives us `Obserable`s